### PR TITLE
oneoff sync of just French

### PR DESCRIPTION
**Pull request by Hamms**
oneoff sync of just French for an Amazon France meeting on Monday

### CODE REVIEW

**Review by maddiedierker — APPROVED** [review]

**Comment by Hamms** [comment]
merging before drone is done to make sure we can get this in today's DTP

File: i18n/locales/fr-FR/dashboard/data.yml [hunk 1]
```diff
@@ -476,10 +476,10 @@ fr:
         MC_2018_loops: 'Minecraft : Voyage Aquatique - Répéter jusqu''à'
         mc_2018_conditionals: 'Minecraft : Voyage Aquatique - Instructions conditionnelles'
         mc_2018_closing: 'Minecraft : Voyage Aquatique - Félicitations'
-        hoc_dance_warmup: Fête dansante - Échauffement
+        hoc_dance_warmup: Dance Party - Warm Up
```
Comment (bethanyaconnor) [inline i18n/locales/fr-FR/dashboard/data.yml:479]: This seems to have reverted to English?

File: dashboard/config/locales/authored_hints.fr-FR.yml [hunk 2]
```diff
@@ -2208,9 +2208,13 @@ fr-FR:
 
           _Ce niveau se termine après 9 mesures._
         challenges: |-
-          **Défis optionnels**
+          "**Optional Challenge**
 
-          Change l’effet de fond à différents moments de la chanson en connectant le nouveau bloc sous les événements <xml><block type="Dancelab_atTimestamp"><title name="TIMESTAMP">???</title><title name="UNIT">"mesures"</title></block></xml>.
+          Change the background effect at different points during the song by connecting the new block under the <xml><block type="Dancelab_atTimestamp"><title name="TIMESTAMP">???</title><title name="UNIT">"measures"</title></block></xml> events.".
```
Comment (maddiedierker) [inline dashboard/config/locales/authored_hints.fr-FR.yml:2213]: (piggy-backing on [bethany's comment](https://github.com/code-dot-org/code-dot-org/pull/32621#discussion_r365428377)) looks like this translation was reverted to English as well

File: i18n/locales/fr-FR/dashboard/data.yml (hunk 1 above)
Comment (Hamms) [reply i18n/locales/fr-FR/dashboard/data.yml:479]: I believe this was intentional; "Dance Party" is a brand and shouldn't be translated. @castro-jorge can you confirm?

File: dashboard/config/locales/authored_hints.fr-FR.yml (hunk 2 above)
Comment (Hamms) [reply dashboard/config/locales/authored_hints.fr-FR.yml:2213]: @castro-jorge looks like TBH translated this to English, for some reason: https://crowdin.com/translate/codeorg/all/enus-fr#2033704

**Pull request merged**